### PR TITLE
Cerd 210 graph 

### PR DESCRIPTION
**Pull request by chrisunl**
+ tests

### CODE REVIEW

File: packages/plot-graph/src/components/plot-graph.ts [hunk 1]
```diff
@@ -23,14 +28,67 @@ export class PlotGraph extends ComponentBase<string> {
...
     private axisSize: number = 25;
 
+    /**
+     * Return d3 line function
+     * 
+     * @param  {any} xScale
+     * @param  {any} yScale
+     * @returns d3.Line
+     */
+    private generateLine(xScale: any, yScale: any): d3.Line<any> {
```
Comment (drochgenius) [inline packages/plot-graph/src/components/plot-graph.ts]: `drawLine` may be a better name.

File: packages/plot-graph/src/unit/plot-graph.spec.ts [hunk 2]
```diff
@@ -11,6 +12,44 @@ describe(`<${tagName}>`, (): void => {
...
+        expect(lineCount).to.equal(3);
+    });
+
+    it('should set equations in response to a slot change event', async (): Promise<void> => {
+        withSnippet('default');
+        const el: PlotGraph = document.querySelector('plot-graph') as any;
+        
+        await el.updateComplete;
+        // @ts-ignore
+        el._onSlotChanged(generateOnSlotChangeEvent(el));
```
Comment (drochgenius) [inline packages/plot-graph/src/unit/plot-graph.spec.ts]: Why don;t you  use `appendChild` to add equations as children, and check if they are picked up correctly?
This seems a bit insufficient to me.

Comment (chrisunl) [reply packages/plot-graph/src/unit/plot-graph.spec.ts]: ok, will do.

File: packages/plot-graph/src/components/plot-graph.ts (hunk 1 above)
Comment (chrisunl) [reply packages/plot-graph/src/components/plot-graph.ts]: Ok, changing it.

**Comment by chrisunl** [comment]
Updated..

**Pull request merged**